server/cmd/dcrdex/main.go

```diff
@@ -119,7 +119,7 @@ func mainCore(ctx context.Context) error {
 		fmt.Printf("Load swapper state from file %q with time stamp %v? (y, n, or enter to abort) ",
 			stateFile.Name, encode.UnixTimeMilli(stateFile.Stamp))
 		scanner := bufio.NewScanner(os.Stdin)
-		scan := make(chan bool)
+		scan := make(chan bool, 1)
```

Comment:
This works either way. I would presume it's more memory efficient to specify the buffer size. Let me know if you know, because I'd like to know.

Reply:
Buffered vs. unbuffered is more about correctness (buffered gives a synchronous send/receive) and performance than memory efficiency.  This channel send is in a goroutine of its own so there's no need for the channel to be buffered as the send can just block until main receives.